hpx-sys/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![doc(html_root_url = "https://github.com/STEllAR-GROUP/hpx-rs")]
 #![allow(bad_style, non_camel_case_types, unused_extern_crates)]
 #![allow(dead_code, unused_imports)]
+#![feature(vec_into_raw_parts)]
```

Comment:
The CI complains about this. PTAL

Reply:
Are we committed to using only safe Rust?

Reply:
Actually, I believe that it doesn't make sense to change to unstable Rust so fast unless we *need* it.

Reply:
The question is if we can keep using stable Rust to compile and continue without the nightly version needed to compile the code, how much more efforts would it need?

Reply:
I agree, with you. Let's fix that before merging.

Reply:
In that case, we do not have access to std:random. for that example, we will need to pull in the rand crate. Is this acceptable?

Reply:
>  Is this acceptable?

I would hope so unless there is something I don't know.

Reply:
Dependency creep, basically.